### ISSUE TITLE
Please add & display dates to resources

### ISSUE THREAD

**Issue by bobaekang** [solidjs/solid-site#161]
While exploring the resources on the website, I noticed that the displayed resources lack any information on their publication dates.

For resources on a rapidly evolving/improving project like Solid, I think it is helpful to show dates--especially Article/Video/Podcast items. Certain topics covered in those resources may stay relevant over time, but information on when they were published still give the readers more useful context. Plus, being able to sort them by publication date would be even better.

**Comment by davedbase** [comment]
Hmmm that's an interesting idea. The only thing is that collecting the publication takes for ~100 resources will be very time consuming.

I'd happily add the functionality into the site if I got help populating the dates :-)

**Comment by bobaekang** [comment]
@davedbase I can help you with that. When do you want it to be called? And what format?

**Comment by davedbase** [comment]
Oh it could just be a timestamp in the JSON data for each resource. We just need published_at as a key. Dates should be in UTC so we can localize it.

**Comment by pheianox** [comment]
In terms of phycology it's not a good idea. People tend to think that a good package is the latest one, but the old ones may be more stable or something like that. That just my opinion.

**Comment by bobaekang** [comment]
@pheianox I agree that the recency of an article/podcast/video/etc. is not the best measure for its relevance, let alone quality. Nonetheless, I think adding date information is better than not since

1. (to my best knowledge,) the current list is not in any meaningful order of relevance/quality either and therefore just as confusing if not more; and
2. coming up with an objective measure for relevance/quality for such resources is very difficult if not impossible.

Plus, order by date is a pretty common practice. So I think it is still an improvement vs. the current list.

**Comment by bobaekang** [comment]
@davedbase Even excluding "Utilities" resources (i.e. libraries and packages), there is at least one resource that does not have a clear "published date" value: a YouTube playlist. Is it acceptable to use "last updated" instead?
